com.unity.render-pipelines.core/Editor/Debugging/DebugUIDrawer.Builtins.cs

```diff
@@ -654,6 +654,80 @@ public override bool OnGUI(DebugUI.Widget widget, DebugState state)
         }
     }
 
+    /// <summary>
+    /// Builtin Drawer for Object Debug Items.
```

Comment:
```suggestion
    /// ImGUI Drawer for Object Debug Items.
```

Reply:
i copy pasted that comment from the other drawers, I can update all of them if you think it's better though

Reply:
Then no worries :)